pom.xml

```diff
@@ -60,7 +60,7 @@
         <connection>scm:git:git@github.com:JanusGraph/janusgraph.git</connection>
         <developerConnection>scm:git:git@github.com:JanusGraph/janusgraph.git</developerConnection>
         <url>git@github.com:JanusGraph/janusgraph.git</url>
-        <tag>HEAD</tag>
+        <tag>0.2.0</tag>
     </scm>
     <properties>
         <janusgraph.compatible.versions>0.1.0,0.1.1</janusgraph.compatible.versions>
```

Comment:
does 0.2.0 need to go here too?

Reply:
no, `janusgraph.compatible.versions` is used for backwards compatibility with previous releases.